engines/bastion_katello/test/products/bulk/products-bulk-action.controller.test.js

```diff
@@ -13,6 +13,10 @@ describe('Controller: ProductsBulkActionController', function() {
         };
         translate = function() {};
         CurrentOrganization = 'foo';
+        GlobalNotification = {
+            setSuccessMessage: function() {},
+            setErrorMessage: function() {},
+        }
```

Comment:
Should we add this to the test mocks module since it's repeated in a lot of places?